extensions/query-store/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-Query Store extension for Azure Data Studio simplifies assists users with troubleshooting query performance in SQL Server and Azure SQL and understanding issues caused by plan changes.
+The Query Store extension for Azure Data Studio simplifies and assists users with troubleshooting query performance in SQL Server and Azure SQL, and with understanding issues caused by plan changes.
```

Comment:
IIRC, it is officially called SQL Database.

![image](https://github.com/microsoft/azuredatastudio/assets/13777222/d6402eac-bd87-4180-9746-72a4efc2212c)